### ISSUE TITLE
When many tabs are open, the client can't keep up with metadata responses

### ISSUE THREAD

**Issue by austinjones** [austinjones/tab-rs#105]
Remove all .unwraps() and .expects() from prod code.

```$ tab tab/
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: Lagged(1)', /Users/austinjones/.cargo/registry/src/github.com-1ecc6299db9ec823/tab-command-0.2.0/src/service/tab_state.rs:40:85```

**Comment by austinjones** [comment]
In this case, the lifeline receiver should be used.  It would log LAGGED.

**Comment by austinjones** [comment]
Resolved by #107
